pkg/security/resolvers/hash/resolver_linux.go

```diff
@@ -223,6 +234,10 @@ func (resolver *Resolver) hash(eventType model.EventType, process *model.Process
 			rootPIDs = w.GetPIDs()
 		}
 	}
+	if process.ContainerID == "" {
+		// try also to open directly the file on the filesystem in case the process was a short lived one
+		rootPIDs = append(rootPIDs, 1)
```

Comment:
Do we need to have this modification in this PR, it changes the behavior for both implementations and could have extra impact.

Reply:
Workaround for short lived processes removed